Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -845,9 +846,9 @@ static bool rareDataChangeRequiresLayout(const StyleRareNonInheritedData& first,
     if (first.counterDirectives != second.counterDirectives)
         return true;
 
-    if (!arePointingToEqualData(first.scale, second.scale)
-        || !arePointingToEqualData(first.rotate, second.rotate)
-        || !arePointingToEqualData(first.translate, second.translate))
+    if (first.scale != second.scale
+        || first.rotate != second.rotate
+        || first.translate != second.translate)
```

Comment:
```suggestion
    if (first.scale != second.scale || first.rotate != second.rotate || first.translate != second.translate)
```
It’s only three, so maybe can be horizontal.